[AIRFLOW-XXXX] Update pydoc of mlengine_operator

dossett opened the pull request:
The maxWorkerCount parameter in mlengine is, somewhat surprisingly, a string:

https://cloud.google.com/ml-engine/reference/rest/v1/projects.jobs#PredictionInput

eladkal commented on airflow/contrib/operators/mlengine_operator.py:
```diff
@@ -135,7 +135,7 @@ class MLEngineBatchPredictionOperator(BaseOperator):
     :param max_worker_count: The maximum number of workers to be used
         for parallel processing. Defaults to 10 if not specified.
-    :type max_worker_count: int
+    :type max_worker_count: string
```
> The maxWorkerCount parameter in mlengine is, somewhat surprisingly, a string:

Actually it make sense. There are two properties `Type `and `Format`.
Some parameters like the maxWorkerCount require both  to determine the data type of the property.  Type is `String `and Format is `INT64` 

The reason for this is explained in the docs:

> The type property indicates the type of the property when its sent in JSON requests and responses (JSON supports a small set of data types, see json.org for details). The format property provides additional information about the underlying type. Properties will always have a type property, but some may also have a format property.
> 
> For example, a 64-bit integer cannot be represented in JSON (since JavaScript and JSON support integers up to 2^53). Therefore, a 64-bit integer must be represented as a string in JSON requests/responses. So the type property will be set to "string", but the format property will be set to "int64" to indicate that it is a 64-bit integer.
> 

https://developers.google.com/discovery/v1/type-format

The pydoc should mention both `Type `and `Format `otherwise users will be confused.

idealy it would be nice to have:
```
:param max_worker_count: ....
:type max_worker_count: string
:format max_worker_count: int
```
But If that is unacceptable how about:
`:type max_worker_count: string  (Format int)
`

dossett replied:
@eladkal I think of either of your suggestions are good.  I don't know enough about pydoc to have an opinion on which is best.

OmerJog replied:
@dossett The 2nd option suggested is supported and we have places where we already use it like: https://github.com/apache/airflow/blob/master/airflow/contrib/operators/bigquery_operator.py#L51  so I say go for this. Maybe even extend the explanation a bit so users will understand  it better,

mik-laj replied:
It seems to me that it is enough to change the type and add a short expalination in the parameter description.  It seems to me that it is enough to change the type and add a comment in the parameter description.

mik-laj reviewed (changes requested):
It is necessary to add information why the value is string

dossett committed:
[AIRFLOW-XXXX] Update pydoc of mlengine_operator

The maxWorkerCount parameter in mlengine is, somewhat surprisingly, a string:

https://cloud.google.com/ml-engine/reference/rest/v1/projects.jobs#PredictionInput
